packages/vite-node/src/cli.ts

```diff
@@ -48,11 +49,12 @@ Usage:
   $ vite-node [options] [files]
 
 Options:
-  -r, --root <path>      ${dim('[string]')} use specified root directory
-  -c, --config <file>    ${dim('[string]')} use specified config file
-  -w, --watch           ${dim('[boolean]')} restart on file changes, similar to "nodemon"
-  -s, --silent          ${dim('[boolean]')} do not emit errors and logs
-  --vue                 ${dim('[boolean]')} support for importing Vue component
```

Comment:
The `--inline-deps` flag is quite long, so I had to align the rest of the help. But nothing changed other than that.